db/models.py

```diff
@@ -50,3 +62,71 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        User,
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        MovieSession,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    order = models.ForeignKey(
+        Order,
+        on_delete=models.CASCADE,
+        related_name="tickets"
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["movie_session", "row", "seat"], name="unique_ticket"
+            )
+        ]
+
+    def clean(self) -> None:
+        if self.row > self.movie_session.cinema_hall.rows or self.row < 1:
+            raise ValidationError({
+                "row": [
+                    f"row number must be in available range: (1, rows): "
+                    f"(1, {self.movie_session.cinema_hall.rows})"
+                ]
+            })
+
+        if (
+                self.seat > self.movie_session.cinema_hall.seats_in_row
+                or self.seat < 1
+        ):
+            raise ValidationError({
+                "seat": [
+                    f"seat number must be in available range: "
+                    f"(1, seats_in_row): "
+                    f"(1, {self.movie_session.cinema_hall.seats_in_row})"
+                ]
+            })
```

Comment:
The validation logic for 'seat' ensures that it is within the valid range. Ensure that this logic is consistent with the business rules and that it does not conflict with any other logic in the application.